File: frontend/StudentManagement.Shared/Dtos/HocSinh/HocSinhResponse.cs
using System;

namespace StudentManagement.Shared.Dtos.HocSinh;

public class HocSinhResponse
{
    public string IdHocSinh { get; set; } = string.Empty;
    public string IdLop { get; set; } = string.Empty;
    public string Ten { get; set; } = string.Empty;
    public string GioiTinh { get; set; } = string.Empty;
    public DateTime? NgaySinh { get; set; }
    public string DiaChi { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
}

File: backend/src/main/java/com/demo/student_management/dto/hocsinh/HocSinhResponse.java
package com.demo.student_management.dto.hocsinh;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDate;

@Data
@AllArgsConstructor
public class HocSinhResponse {
    private String idHocSinh;
    private String idLop;
    private String ten;
    private String gioiTinh;
    private LocalDate ngaySinh;
    private String diaChi;
    private String email;
}